sdk-testing/pom.xml

```diff
@@ -0,0 +1,132 @@
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.dapr</groupId>
+    <artifactId>dapr-sdk-parent</artifactId>
+    <version>1.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>dapr-sdk-testing</artifactId>
+  <packaging>jar</packaging>
+  <version>1.2.0-SNAPSHOT</version>
+  <name>dapr-sdk-testing</name>
+  <description>Testing utilities for applications using Dapr SDK.</description>
+
+  <repositories>
+    <repository>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>jcenter</id>
+      <name>jcenter</name>
+      <url>https://jcenter.bintray.com/</url>
```

Comment:
JCenter has been [closed down](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) I believe. I would not expect new references to be introduced (and I would even expect old references to disappear someday - but that should be a separate ticket, probably?)

Reply:
Yes, that needs to be addressed. Too bad. We had to migrate to jcenter due to another mirror being closed down before. It seems to be a thing now for public Maven repositories.

Reply:
Tracking here: https://github.com/dapr/java-sdk/issues/608